internal/util/csiconfig.go

```diff
@@ -106,6 +107,21 @@ func GetRadosNamespace(pathToConfig, clusterID string) (string, error) {
 	return cluster.RBD.RadosNamespace, nil
 }
 
+// GetCephFSRadosNamespace returns the namespace for the given clusterID.
+// If not set, it returns the default value "csi".
+func GetCephFSRadosNamespace(pathToConfig, clusterID string) (string, error) {
+	cluster, err := readClusterInfo(pathToConfig, clusterID)
+	if err != nil {
+		return "", err
+	}
+
+	if cluster.CephFS.RadosNamespace == "" {
+		return fsutil.RadosNamespace, nil
```

Comment:
try to avoid importing fsutil in this standard util as it can create a circular dependency problem in the future.